.github/workflows/prerelease.yaml

```diff
@@ -0,0 +1,28 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: pre-release
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Just for understanding... Couldn't we remove push here and get rid of the job guard?

Reply:
I think so. In a related note, is there any reason not wanting prerelease tests to run with every push or nightly?

Reply:
I think the reason is there is no concept of workflows that are allowed to fail with GitHub Actions.  We will want to be able to merge changes between releases that will cause the pre-release test to fail.

Reply:
I pushed a change removing the `push` from this.  I'm not sure what the original intent of having it here.  I mostly copied this file from the moveit repo.